source/less/blocks/advantages.less

```
@@ -46,16 +123,64 @@
   .advantages__item {
     min-height: 280px;
     box-sizing: border-box;
+    margin-bottom: 0;
+    padding: 0;
     padding-top: 151px;
-    padding-left: 0;
+  }
+
+  .advantages__item::before {
+    width: 80px;
+    height: 80px;
+    top: 40px;
+  }
+
+  .advantages__item:nth-child(1)::after {
+    top: 67px;
+    left: 27px;
+  }
+
+  .advantages__item:nth-child(2)::after {
+    top: 63px;
+    left: 25px;
+  }
+
+  .advantages__item:nth-child(3)::after {
+    top: 65px;
+    left: 28px;
+  }
+
+  .advantages__item:nth-child(4)::after {
+    top: 63px;
+    left: 24px;
+  }
+
+  .advantages__item-number {
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: block;
+    margin: 0;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 280px;
+    line-height: 280px;
+    color: #f2f2f2;
+  }
```

Это в продолжении про стилизацию списка.
```suggestion
  .advantages__item::marker {
    position: absolute;
    top: 0;
    right: 0;
    display: block;
    margin: 0;
    font-style: normal;
    font-weight: 400;
    font-size: 280px;
    line-height: 280px;
    color: #f2f2f2;
  }
```
Убрать `.` после числа чуть сложнее, но тоже возможно, поищи про CSS list counters